cypher/cypher-docs/src/docs/dev/schema.asciidoc

```diff
@@ -6,24 +6,27 @@
 This section explains how to work with an optional schema in Neo4j in the Cypher query language.
 --
 
-Neo4j 2.0 introduced an optional schema for the graph, based around the concept of labels.
+Neo4j provides an optional schema for the graph, based around the concept of labels.
```

Comment:
Many customers consider the labels and relationship types the _schema_ as they are a bit like table names. And then constraints also fit into this thinking, for example property existance constrants are like requiring the table to have certain columns. When we do property type constraints that also fits into schema. But indexes do not.

So we have a clash of mindsets, how customers see _schema_ versus how we internally see it, particularly around the concept of labels, which are very soft in Neo4j and very unlike 'schema'. But I believe in some cases, like the 3.1 security project, we did give some credence to the idea that labels are schema, separating the 'write' user into one that could 'add labels, relationship types and data' from another that could only 'add data'.

Reply:
Not sure what is needed here? Please clarify. I, for one, disliked the sentence from the get-go and after your comments, have elected to remove it altogether.